### ISSUE TITLE
Missing page to iron-a11y-keys

### ISSUE THREAD

**Issue by philippw** [webcomponents/webcomponents.org#1073]
On webcomponents.org there's currently no page for the [iron-a11y-keys](https://github.com/PolymerElements/iron-a11y-keys): [this link](https://elements.polymer-project.org/elements/iron-a11y-keys) is broken.

**Comment by samuelli** [comment]
Thanks for reporting!

Fixed: https://www.webcomponents.org/element/PolymerElements/iron-a11y-keys
